docs/strategies/hedging.md

```diff
@@ -147,6 +337,102 @@ With this configuration, the hedging strategy:
 - Initiates a maximum of `4` executions. This includes initial action and an additional 3 attempts.
 - Allows the first two executions to proceed in parallel, while the third and fourth executions follow the fallback mode.
 
+#### Dynamic: happy path sequence diagram
+
+The hedging strategy triggers and switches between modes due to the usage of `DelayGenerator`. It succeeds because the last request succeeds.
+
+```mermaid
+sequenceDiagram
+    actor C as Caller
+    participant P as Pipeline
+    participant H as Hedging
+    participant DG as DelayGenerator
+    participant HUC as HedgedUserCallback
+
+    C->>P: Calls ExecuteAsync
+    P->>H: Calls ExecuteCore
+    activate H
+
+    Note over H: Parallel mode
+    par
+    H -> DG: Gets delay
```

Comment:
We should have all these "Gets delay" lines be like the others with an arrow each way and with the delay value being returned, rather than just a single horizontal line.